src/cli.ts

```diff
@@ -47,4 +49,5 @@ format(program);
 monitor(program);
 uuid(program);
 kfc(program);
+crd(program);
```

Comment:
We considered printing a brief message that this feature is in `alpha` to users. Where would be the best place to log it? Here or within `cli/crd/index.ts`?

Reply:
yes sounds good